src/server/js/server.js

```diff
@@ -1,18 +1,17 @@
 require('babel-core/register');
-import { graphqlExpress, graphiqlExpress } from 'graphql-server-express';
 import bodyParser from 'body-parser';
+import express from 'express';
+import { readdirSync, statSync } from 'fs';
+import { graphqlExpress, graphiqlExpress } from 'graphql-server-express';
+import path from 'path';
 
-const PORT = process.env.PORT || '8080';
-
-const express = require('express');
-const path = require('path');
-const { readdirSync, statSync } = require('fs');
-const { join } = require('path');
-
+import passport from './passport';
+import routes from './routes';
```

Comment:
nothing changed in this section, just using es6 imports and moved stuff around a bit